lib/perl/Genome/Sys.pm

```diff
@@ -940,7 +940,16 @@ sub set_gid {
 
 sub gidgrnam {
     my $group_name = shift;
-    (getgrnam($group_name))[2];
+
```

Comment:
If it's the new LSF job that you created to wait for all the others, then that makes sense!  If it's something the test was already `bsub`bing, which found this different problem, then I believe it's better as a different PR.  (At this point it's already in here, so I wouldn't worry about changing it!)